packages/c/cryptopp/xmake.lua

```diff
@@ -17,7 +19,20 @@ package("cryptopp")
     on_install("windows", "macosx", "linux", "bsd", "iphoneos", function (package)
         local cryptopp_cmake = package:resourcedir("cryptopp_cmake")
         os.cp(path.join(cryptopp_cmake, "*", "CMakeLists.txt"), ".")
-        os.cp(path.join(cryptopp_cmake, "*", "cryptopp-config.cmake"), ".")
+        local version = package:version()
+        
+        if (version:ge("8.7")) then
+            if (package:config("shared")) then
+                print("ERROR: CryptoC++ 8.7+ does not support shared libraries")
```

Comment:
@waruqi I don't know if this is appropriate, let me know if I should handle this differently